hello.rb

```diff
@@ -1,2 +1,3 @@
 #!/usr/bin/env ruby
-puts "hello world"
+name = ARGV[0]
```

Comment:
what happens if someone doesnt supply arguments? Shouldn't we handle this error?

Reply:
yo, changed that